commons/com.b2international.index/.classpath

```diff
@@ -51,13 +44,25 @@
 	<classpathentry exported="true" kind="lib" path="lib/reindex-client-7.17.16.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/scripting-painless-spi-7.17.16.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/ssl-config-7.17.16.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/t-digest-3.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/transport-7.17.16.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/transport-netty4-client-7.17.16.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" output="target/classes" path="src">
 		<attributes>
 			<attribute name="optional" value="true"/>
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.m2e.MAVEN2_CLASSPATH_CONTAINER">
```

Comment:
Please remove the MAVEN2 classpath containers and avoid running the Alt + F5 Maven project update with the update project configuration from `pom.xml` option.